corehq/apps/cowin/models.py

```diff
@@ -0,0 +1,2 @@
+# import so that signals are set on app load
+from .signals import *  # noqa
```

Comment:
To repeat the feedback I got on my last Repeater PR, we use [django.dispatch.receiver()](https://docs.djangoproject.com/en/2.2/topics/signals/#connecting-receiver-functions) and [AppConfig](https://docs.djangoproject.com/en/2.2/ref/applications/#configuring-applications) to set up signals.

Here's the commit of how I responded to my PR feedback, as an example: https://github.com/dimagi/commcare-hq/commit/52b72fca3855f285c4dcba2110d01b71ce5d4c09

Reply:
https://github.com/dimagi/commcare-hq/pull/29949/commits/db3688d46a93a9814fbe84b560b576cee58cce1d

Interesting. Thanks for sharing that. Also, I searched a bit in code base and saw that i could just do it in the __init__ file, for example as done in users app, so avoided the apps file.